releasenotes/notes/0.45/rust-1.64-add4a4a8e5ce9786.yaml

```
@@ -1,5 +1,5 @@
 ---
-upgrade:
+upgrade_misc:
   - |
     Building Qiskit from source now requires a Rust compiler compatible with language version 1.64,
     up from 1.61.
```

I'm not sure if this is needed or not. I just thought up from 1.61 might be confusing for some people.

```suggestion
    Building Qiskit from source now requires a Rust compiler compatible with language version 1.64.
    This has been increased from the previous minimum supported Rust version of 1.61 for building
     earlier versions of Qiskit.
```